### PR TITLE
Added round_nearest function

### PR DESCRIPTION
**Pull request by lauraselby**
Added the round_nearest function to the formatting file.

I've added the documentation and some tests too.

### CODE REVIEW

File: tests/testthat/test_formatting.R [hunk 1]
```diff
@@ -19,3 +19,21 @@ test_that("Example numerical known answer", {
...
+test_that("Input as expected", {
+  expect_error(round_nearest(12, "ten"), "n must be a number")
+})
+
+test_that("Input as expected", {
+  expect_error(round_nearest("twelve", 10), "x must be a number")
+})
+
+test_that("Example numerical known answer", {
+  expect_equal(round_nearest(1337,5), "1335")
```
Comment (adamrobinson361) [inline tests/testthat/test_formatting.R]: Travis was playing up a bit but should be fixed now. These 2 latter tests fail as the output of your function is a double not a character. Can you change to 1335 and 1340

File: tests/testthat/test_formatting.R [hunk 2]
```diff
@@ -19,3 +19,21 @@ test_that("Example numerical known answer", {
...
+})
+
+test_that("Example character known answer", {
+  expect_equal(round_nearest("1337",10), "1340")
```
Comment (adamrobinson361) [inline tests/testthat/test_formatting.R]: Heres the other one

**Comment by lauraselby** [comment]
Thanks Adam - updated to correct the errors in two tests

**Comment by adamrobinson361** [comment]
All looks good - i've added a link in documentation and updated version

**Pull request merged**